docs/src/app/components/pages/components/progress.jsx

```diff
@@ -16,7 +16,7 @@ const ProgressPage = React.createClass({
   componentDidMount() {
     let self = this;
 
-    let id = window.setInterval(() => {
+    this.timer = window.setInterval(() => {
```

Comment:
What about adding it at the top like here https://github.com/callemall/material-ui/blob/master/src/snackbar.jsx#L24?